pages/6_Change_over_Time.py

```diff
@@ -16,7 +16,6 @@
 
```

Comment:
I get this error when I run this page with the non-multimodel version of the brakepad dataset
![image](https://github.com/sibyl-dev/sibylapp2/assets/14080838/07dc66e1-647a-4528-b4ac-750fd160e920)

Reply:
The page only makes sense when there are multiple models -- I think we should disable the page when only dealing with one

Reply:
Yes, right now we disable it when there are only single rows I believe, we should do the same for single models (just so long as we don't have an ugly error)